Wikipedia/Code/TalkPageTopicComposeViewController+TalkPageFormattingToolbar.swift

```diff
@@ -0,0 +1,118 @@
+import Foundation
+import CocoaLumberjackSwift
+import WMF
+
+extension TalkPageTopicComposeViewController: TalkPageFormattingToolbarViewDelegate {
+
+    func didSelectBold() {
+        bodyTextView.addOrRemoveStringFormattingCharacters(formattingString: "'''")
+    }
+
+    func didSelectItalics() {
+        bodyTextView.addOrRemoveStringFormattingCharacters(formattingString: "''")
+    }
+
+    func didSelectInsertLink() {
```

Comment:
I know you already mentioned this so just commenting for tracking purposes. It'd be good to DRY up some of the logic in these delegate methods between new topic and new reply. Let me know if you want any thoughts on approaches for that.